sdk/nodejs/npm/manager.go

```diff
@@ -82,17 +81,12 @@ func Install(ctx context.Context, packagemanager PackageManagerType, dir string,
 	}
 
 	// Ensure the "node_modules" directory exists.
-	// NB: This is only appropriate for certain package managers.
-	//     Yarn with Plug'n'Play enabled won't produce a node_modules directory,
-	//     either for Yarn Classic or Yarn Berry.
```

Comment:
Is this comment no longer relevant because of the search up?

Reply:
Hmmm I guess it's still true for yarn pnp, since I think that doesn't create a `node_modules` folder, but a `.pnp.cjs` file. I don't think we'll support that anytime soon though, since I think it requires you to execute via yarn (so it can setup its import loader). 🤷

Reply:
If we do add support, this will immediately fail, so it's not hard to figure out what we do in that case.

Reply:
Ah apologies I missed the fact that we don't support them -- this comment implied to me that did and we hadn't something that didn't work, as opposed to "if we try to do X, it'll fail". All good!